test-runner-poc/environments.js

```diff
@@ -60,7 +60,7 @@ class ReplicaSetEnvironment extends EnvironmentBase {
       return new core.ReplSet([{ topologyHost, topologyPort }], options);
     };
 
-    this.nodes = generateNodesArray(parsedURI.hosts, genReplsetConfig);
+    //this.nodes = generateNodesArray(parsedURI.hosts, genReplsetConfig);
```

Comment:
Does everything work with this commented out?

Reply:
yes - we are currently not calling `generateNodesArray` anywhere else, and the tests pass

Reply:
Should I remove the comment?

Reply:
yep!